src/App.tsx

```diff
@@ -1,26 +1,175 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import cn from 'classnames';
+import React, { useEffect, useMemo, useRef, useState } from 'react';
+
+import {
+  USER_ID,
+  addTodo,
+  getTodos,
+  filterTodos,
+  deleteTodo,
+} from './api/todos';
 
-const USER_ID = 0;
+import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { Status } from './types/Status';
+import { TodoFilter } from './components/TodoFilter';
+import { TodoItem } from './components/TodoItem';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
-
-  return (
-    <section className="section container">
-      <p className="title is-4">
-        Copy all you need from the prev task:
-        <br />
-        <a href="https://github.com/mate-academy/react_todo-app-loading-todos#react-todo-app-load-todos">
-          React Todo App - Load Todos
-        </a>
-      </p>
-
-      <p className="subtitle">Styles are already copied</p>
-    </section>
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [newTodoTitle, setNewTodoTitle] = useState('');
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [errorMsg, setErrorMsg] = useState('');
```

Comment:
u can create enum for all the error. But up to u
```suggestion
  const [errorMsg, setErrorMsg] = useState<ErrorType>(ErrorType.Default);
```